packages/devel/glibc/package.mk

```diff
@@ -3,8 +3,8 @@
 # Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="glibc"
-PKG_VERSION="2.31"
-PKG_SHA256="9246fe44f68feeec8c666bb87973d590ce0137cca145df014c72ec95be9ffd17"
+PKG_VERSION="2.32"
```

Comment:
glibc 2.32 removed `--enable-obsolete-rpc`, which is needed for NFS booting. tirpc and rpcsvc-proto(?) need to be packaged and tested. @HiassofT might use a NFS set up to test.

Reply:
we've switched to libtirpc a long time ago.

Quick NFS/rpc tests look good here

Reply:
the last open point, was the nfs-boot.
this wasn't working easly. When this is now working, we can drop `obsolete-rpc`.

Reply:
just re-tested with another clean build and obsolete-rpc dropped and it worked fine

@heitbaum can you remove these lines? https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/devel/glibc/package.mk#L38-L41
```
# busybox:init needs it
# testcase: boot with /storage as nfs-share (set cmdline.txt -> "ip=dhcp boot=UUID=2407-5145 disk=NFS=[nfs-share] quiet")
PKG_CONFIGURE_OPTS_TARGET+=" --enable-obsolete-rpc"

```

Reply:
@HiassofT - rebased with requested lines removed